share/spice/nxt_account/nxt_account.js

```diff
@@ -0,0 +1,45 @@
+(function (env) {
+    "use strict";
+
+    var formatNxt = function(amount) {
+        return (amount / 100000000)+" NXT"
+    };
+
+    env.ddg_spice_nxt_account = function(api_result) {
+
+        if (!api_result) {
+            return Spice.failed('nxt_account');
+        }
+
+        Spice.add({
+            id: "nxt_account",
+            name: "Nxt Blockchain",
```

Comment:
I don't think Software is quite right. We use "Bitcoin Wallet" for the related Bitcoin IA, I think "Nxt Wallet", or switching both to "Currency" seems OK.